worlds/hk/__init__.py

```diff
@@ -252,21 +252,21 @@ def _add(item_name: str, location_name: str):
             if item_name in junk_replace:
                 item_name = self.get_filler_item_name()
 
-            item = self.create_item(item_name)
-
             if location_name == "Start":
                 if item_name in randomized_starting_items:
                     if item_name == "Focus":
                         self.create_location("Focus")
                         unfilled_locations += 1
-                    pool.append(item)
+                    pool.append(self.create_item(item_name))
                 else:
-                    self.multiworld.push_precollected(item)
+                    self.multiworld.push_precollected(self.create_item(item_name))
```

Comment:
Create_location uses `vanilla: bool` so probably one of the 2 should be inverted

Reply:
not sure what this comment is referring to, especially with the code line it is pointing to

Reply:
Idk how GitHub screwed it up so bad, referring to the signature for _add vs create_location, one uses randomized the other uses vanilla

Reply:
yea that already existed so i just ran with it, both use vanilla but _add inverts randomized before using it as vanilla